draft-ietf-ice-rfc5245bis.xml

```diff
@@ -1585,15 +1585,19 @@ pairs, called the check list for that data stream.
 <t>
 In order to limit the attacks described in <xref
 target="sec-ice-hammer"/>, an ICE agent MUST limit the total number of
-connectivity checks the agent performs across all check lists to a
-specific value, and this value MUST be configurable. A default of 100
-is RECOMMENDED. This limit is enforced by discarding the
-lower-priority candidate pairs until there are less than 100. It is
-RECOMMENDED that a lower value be utilized when possible, set to the
-maximum number of plausible checks that might be seen in an actual
-deployment configuration. The requirement for configuration is meant
-to provide a tool for fixing this value in the field if, once
-deployed, it is found to be problematic.
+connectivity checks the agent performs across all check lists in the check list set 
+to a specific value. The default limit value of connectivity checks for the check 
+list set is 100, but the value MUST be configurable. The limit is enforced by, 
+within in each check list, discarding lower-priority candidate pairs until 
+the number of candidate pairs in the check list set is smaller than the limit value.
+The discarding SHOULD be done evenly so that the number of remaining number of 
+connection checks in each check list are equal. 
+</t>
+<t>
+It is RECOMMENDED that a lower limit value than the default is picked when possible, 
```

Comment:
I use "pick" elsewhere, in order to avoid confusing with selected. There are only 5 instances of "chose" throughout the document - and perhaps those can be changed to "pick" also.